cmd/influx_stress/influx_stress.go

```diff
@@ -12,6 +12,7 @@ import (
 var (
 	config     = flag.String("config", "", "The stress test file")
 	cpuprofile = flag.String("cpuprofile", "", "Write the cpu profile to `filename`")
+	db         = flag.String("db", "", "name of the database where the stress test will take place")
```

Comment:
"take place" isn't that clear to me. How about:

"target database within test system for write load".